package.json

```diff
@@ -91,7 +93,8 @@
         "mocha": "6.1.4",
         "sinon": "7.2.5",
         "yeoman-assert": "3.1.1",
-        "yeoman-test": "1.9.1"
+        "yeoman-test": "1.9.1",
+        "@openapitools/openapi-generator-cli": "0.0.9-3.3.4"
```

Comment:
Better to put 0.0.9-4.0.2

See https://github.com/jhipster/generator-jhipster/pull/9925

Reply:
updated to 0.0.14-4.0.2